app/Jobs/SendTravelAssignmentReminder.php

```diff
@@ -25,16 +25,10 @@ class SendTravelAssignmentReminder implements ShouldQueue, ShouldBeUnique
     /**
      * Create a new job instance.
      */
-    public function __construct(private readonly TravelAssignment $assignment)
+    public function __construct(private readonly TravelAssignment $assignment, int $delay = 48)
     {
         $this->queue = 'email';
-        $this->delay = now()->addHours(48)->hour(10)->startOfHour()->addMinutes(random_int(10, 50));
-
-        if ($this->delay->dayOfWeek === 5) {
-            // do not send reminders on thursdays to reduce the chance of user
```

Comment:
Is this no longer relevant?

Reply:
It doesn't seem to work, I don't want to troubleshoot it, and there isn't actually that much downtime (usually)